script.js

```diff
@@ -0,0 +1,85 @@
+const form = document.getElementById('form');
+let books = JSON.parse(localStorage.getItem('books'));
+// add class
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+
+  newBook() {
+    const { title, author, id } = this;
+    const bookList = { title, author, id };
+    books = JSON.parse(localStorage.getItem('books'));
+    if (title === '' || author === '') {
+      document.getElementById('error').innerHTML = 'Required';
+    } else if (books !== null) {
+      books.push(bookList);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+    } else {
+      books = [];
+      books.push(bookList);
+      localStorage.setItem('books', JSON.stringify(books));
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+  }
+
+  // remove
+  remove() {
+    const { id } = this;
+    books = books.filter((book) => {
+      if (book.id === id) {
+        return false;
+      }
+      return true;
+    });
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+}
+
+// books
+function displayBooks(title, author, id) {
```

Comment:
- [ ] [OPTIONAL] Kindly change the `displayBooks` syntax to es6 syntax in order to make your code cleaner. so it will be like:
```
const displayBooks = (title, author, id) => {
}
```